base/Base.jl

```diff
@@ -6,10 +6,19 @@ using Core.Intrinsics, Core.IR
 
 # to start, we're going to use a very simple definition of `include`
 # that doesn't require any function (except what we can get from the `Core` top-module)
-const _included_files = Array{Tuple{Module,String},1}(Core.undef, 1)
+# start this big so that we don't have to resize before we have defined how to grow an array
+const _included_files = Array{Tuple{Module,String},1}(Core.undef, 50)
```

Comment:
```suggestion
const _included_files = Array{Tuple{Module,String},1}(Core.undef, 400)
```
For performance reasons :P

But also in case we learn how to grow an array later.